<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketEditDialog</class>
 <widget class="QDialog" name="TicketEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="brandLabel">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>brand</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="modelLabel">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="regIDLabel">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>regid</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="legendLabel">
       <property name="text">
        <string>White - free time slots,  Gray - unavailable time slots, Yellow -  time slots selected for this repair</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignHCenter">
      <widget class="QTableWidget" name="calendar">
       <property name="minimumSize">
        <size>
         <width>780</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>780</width>
         <height>300</height>
        </size>
       </property>
       <row>
        <property name="text">
         <string>8:00-9:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>9:00-10:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>10:00-11:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>11:00-12:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>12:00-13:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>13:00-14:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>14:00-15:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>15:00-16:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>16:00-17:00</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Mon</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tue</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Wed</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Thu</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fri</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sat</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sun</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="3" column="0"/>
       <item row="4" column="0"/>
       <item row="5" column="0"/>
       <item row="6" column="0"/>
       <item row="7" column="0"/>
       <item row="8" column="0"/>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="chooseEmpLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Choose employee to edit repair schedule:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="employees">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="costLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>26</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Estimated labour cost:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="costEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="partsLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Parts used for repair:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="partsList"/>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addPartsButton">
         <property name="text">
          <string>Add parts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removePartsButton">
         <property name="text">
          <string>Remove parts</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>13</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ticket status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="statusBox">
       <item>
        <property name="text">
         <string>created</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>in progress</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>done</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>closed</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="descriptionLabel">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Issue description:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="descriptionEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="estAcceptedLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Accepted by client:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="acceptedBox">
       <item>
        <property name="text">
         <string>no</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yes</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="clientPaidLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Price paid by client:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="clientPaidEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
